code/datums/martial/wrestling.dm

```diff
@@ -17,6 +17,18 @@
 	var/datum/action/strike/strike = new/datum/action/strike()
 	var/datum/action/drop/drop = new/datum/action/drop()
 
+/datum/martial_art/wrestling/teach(mob/living/carbon/human/H, make_temporary)
+	. = ..()
+	ADD_TRAIT(H,TRAIT_STUNRESISTANCE,type)
+	ADD_TRAIT(H,TRAIT_PUSHIMMUNE,type)
+	ADD_TRAIT(H,TRAIT_IGNOREDAMAGESLOWDOWN,type)
```

Comment:
Jesus christ

Reply:
stun resistance might be a bit much i agree

Reply:
i didnt click stun resistance, i clicked the ignore damage slowdown